docs/guides/basic-database.rst

```diff
@@ -1,194 +1,106 @@
-    15:31:30 About to create 3 backend threads
-    15:39:55 [gMySQLbackend] MySQL connection succeeded
-    15:39:55 [gMySQLbackend] MySQL connection succeeded
-    15:39:55 [gMySQLbackend] MySQL connection succeeded
+Make sure no error is reported, and use ``systemctl status pdns`` to make sure PowerDNS was started correctly.
 
-In a different shell, a sample query sent to the server should now
-return quickly *without* data:
+A sample query sent to the server should now return quickly *without* data:
 
 .. code-block:: shell
 
-    $ dig +short www.example.com @127.0.0.1  # should print nothing
-
-.. warning::
-  When debugging DNS problems, don't use ``host``. Please use
-  ``dig`` or ``drill``.
-
-And indeed, the output in the first terminal now shows:
+     dig a www.example.com @127.0.0.1 
 
-::
+    ; <<>> DiG 9.10.3-P4-Debian <<>> a www.example.com @127.0.0.1
+    ;; global options: +cmd
+    ;; Got answer:
+    ;; ->>HEADER<<- opcode: QUERY, status: REFUSED, id: 40870
+    ...
 
-    Mar 01 16:04:42 Remote 127.0.0.1 wants 'www.example.com|A', do = 0, bufsize = 1680: packetcache MISS
+.. warning::
+  When debugging DNS problems, don't use ``host``. Please use ``dig`` or ``drill``.
 
-Now we need to add some records to our database (in a separate shell):
+Note the ``REFUSED`` status - this is the code most name servers use to indicate they do not know about a domain.
 
-::
+Now, let's add a zone and some records:
 
-    # mysql pdnstest
-    mysql> INSERT INTO domains (name, type) values ('example.com', 'NATIVE');
-    INSERT INTO records (domain_id, name, content, type,ttl,prio)
-    VALUES (1,'example.com','localhost admin.example.com 1 10380 3600 604800 3600','SOA',86400,NULL);
-    INSERT INTO records (domain_id, name, content, type,ttl,prio)
-    VALUES (1,'example.com','dns-us1.powerdns.net','NS',86400,NULL);
-    INSERT INTO records (domain_id, name, content, type,ttl,prio)
-    VALUES (1,'example.com','dns-eu1.powerdns.net','NS',86400,NULL);
-    INSERT INTO records (domain_id, name, content, type,ttl,prio)
-    VALUES (1,'www.example.com','192.0.2.10','A',120,NULL);
-    INSERT INTO records (domain_id, name, content, type,ttl,prio)
-    VALUES (1,'mail.example.com','192.0.2.12','A',120,NULL);
-    INSERT INTO records (domain_id, name, content, type,ttl,prio)
-    VALUES (1,'localhost.example.com','127.0.0.1','A',120,NULL);
-    INSERT INTO records (domain_id, name, content, type,ttl,prio)
-    VALUES (1,'example.com','mail.example.com','MX',120,25);
+.. code-block:: shell
 
-.. warning::
-  Host names and the MNAME of a :ref:`types-soa`
-  records are NEVER terminated with a '.' in PowerDNS storage! If a
-  trailing '.' is present it will inevitably cause problems, problems that
-  may be hard to debug.
+    $  create-zone example.org ns1.example.com
+    Creating empty zone 'example.com'
+    Also adding one NS record
+    $ pdnsutil add-record example.com '' MX '25 mail.example.com'
```

Comment:
```suggestion
    sudo -u pdns pdnsutil add-record example.com '' MX '25 mail.example.com'
```

Reply:
I feel removing the `$` here makes the block hard to read. I now also have doubts about the other suggestions I committed for that :)